packages/protocol/contracts/L1/TaikoEvents.sol

```diff
@@ -17,7 +17,8 @@ abstract contract TaikoEvents {
         bytes32 parentHash,
         bytes32 blockHash,
         bytes32 signalRoot,
-        address prover
+        address prover,
+        uint32 parentGasUsed
```

Comment:
hmmm since `parentHash` field is in this event, not sure if its necessary to compare the `parentGasUsed` in client again?

Reply:
@Brechtpd what do you think? you mentioned in the taiko-client PR to compare both

Reply:
I think it's OK to have `parentGasUsed` in the event.

Reply:
> hmmm since `parentHash` field is in this event, not sure if its necessary to compare the `parentGasUsed` in client again?

The prover can submit any `parentGasUsed` value, so if the prover did not submit the correct value for it to the smart contract the proof that was submitted will be in the wrong fork choice so we still need people to submit a valid proof with both the correct `parentHash` and `parentGasUsed` for the block to be able to get verified. Or do I miss something?